templates/service.yaml

```diff
@@ -32,7 +32,7 @@ spec:
   ports:
     - port: {{ .Values.service.port }}
       protocol: TCP
-      name: {{ .Values.service.name }}
+      name: http-{{ .Values.service.name }}
```

Comment:
Is there a different prefix for `https`? The registry can be configured to serve via HTTP or HTTPS, so if there is a different prefix for https, we'll want to take that into account here.

Reply:
I hopefully fixed this by checking if `{{ .Values.tlsSecret }}` is set.

Reply:
That seems reasonable to me. :+1: